### ISSUE TITLE
Duplicate ids in '10: Numeric Annotation Glyphs'

### ISSUE THREAD

**Issue by ghost** [mliebelt/pgn-spec-commented#18]
The following section in '10: Numeric Annotation Glyphs'
...
93 	Black has good bishop placement
84 	White has poor rook placement
85 	Black has poor rook placement
86 	White has good rook placement
87 	Black has good rook placement
98 	White has poor queen placement
...

has the identical number as the section

...
84 	White has a very strong pawn structure
85 	Black has a very strong pawn structure
86 	White has poor knight placement
87 	Black has poor knight placement
...

In other words, the glyph numbers are not unique, the numbers 84, 85, 86 and 87 are used twice

**Comment by mliebelt** [comment]
I will check that, and fix it then. Thank you for bringing that to my attention.

**Comment by mliebelt** [comment]
Interestingly, it was wrong for all the resources. And no one noticed that in the past 30 years or so ... I have fixed it in all the resources I own, but I cannot change it in the original location.
